src/main/kotlin/de/livepoll/api/util/AccountListener.kt

```diff
@@ -29,15 +32,34 @@ class AccountListener: ApplicationListener<OnCreateAccountEvent> {
         accountService.createVerificationToken(user, token)
 
         val recipientAddress = user.email
-        val subject = "Account Confirmation Live-Poll"
-        val confirmationUrl = event.appUrl + "/v1/account/confirm?token=$token"
-        val message = "Please confirm your email: "
-
-        javaMailSender.send(SimpleMailMessage().apply {
-            setTo(recipientAddress)
-            setSubject(subject)
-            setText("$message\r\n$serverUrl$confirmationUrl")
-        })
-    }
+        val subject = "Your Live-Poll registration"
+        val confirmationUrl = "$frontendUrl/activate/$token"
 
+        val mimeMessage: MimeMessage = javaMailSender.createMimeMessage()
+        val helper = MimeMessageHelper(mimeMessage, true)
+        helper.setFrom("noreply@live-poll.de")
+        helper.setTo(recipientAddress)
+        helper.setSubject(subject)
+        helper.setText(
+                "<html>"
+                        + "<body>"
+                        + "<img src='cid:logo' style='float:top;width:150px;height:150px;'/>"
+                        + "<div style='margin:1rem'/>"
+                        + "<div>Dear ${user.username},"
+                        + "<div>thank you for using Live-Poll."
+                        + "<div style='margin:1rem'/>"
+                        + "<div>Live-Poll is an open-source live-polling application that you can use totally free, no matter if you are a private person, school, university, society, small or big business etc. Our idea arose from the lack of free live voting/polling software on the Internet that has a nice user flow and is easy to use.</div>"
+                        + "<div style='margin:1rem'/>"
+                        + "<div>Please confirm your registration by clicking <a href='$confirmationUrl'>here</a>.</div>"
+                        + "<div style='margin:1rem'/>"
+                        + "<div>Best regards,</div>"
+                        + "<div>Live-Poll</div>"
+                        + "</div>"
+                        + "</div></body>"
+                        + "</html>", true)
```

Comment:
Nice, love it 😍

Reply:
In Mail for Windows, the image is a bit huge, but I think that is not that important now.